demo/components/victory-axis-demo.jsx

```diff
@@ -74,7 +74,11 @@ export default class App extends React.Component {
             label="time axis"
             padding={{left: 10, right: 80}}
             scale="time"
-            style={{grid: {stroke: "black", strokeWidth: 1}}}
+            events={{axis: {onClick: (evt) => alert(`x: ${evt.clientX}, y: ${evt.clientY}`)}}}
```

Comment:
I'd like to think of something simple to show interaction that doesn't use `alert`. Besides being cruddy, browsers now will inject a "prevent this web page from showing more alerts?" checkbox that isn't what we want for showcasing victory event functionality.

Even having a simple component living side by side with the demo component that renders the text `Clicked coordinates: X, Y` would be better.

I realize this is extra work, but I think we either should spend the time, or we don't bother showing interactivity in the demos until we have beefier examples like tooltips.